index.html

```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+		
+		<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Hi, please replace "template: make a copy" with your name for both card start and card end

Reply:
Hi, done)